### PR TITLE
New package: m17n-db-1.7.0

### PR DESCRIPTION
**Pull request by hyonhyoro**

### CODE REVIEW

File: srcpkgs/m17n-db/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'm17n-db'
+pkgname=m17n-db
+version=1.7.0
+revision=1
+noarch=yes
+only_for_archs="aarch64 armv7hf i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/m17n-db/template]: why?

Comment (Vaelatern) [inline srcpkgs/m17n-db/template]: No, really, why? Especially having both `only_for_archs` and `noarch`. Why?

File: srcpkgs/m17n-db/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+homepage="http://www.nongnu.org/m17n/"
+distfiles="${NONGNU_SITE}/m17n/${pkgname}-${version}.tar.gz"
+checksum=a2ba9f80161433d5c06e57915a9cd51f26c6df4a8909723f952cdbb9b48508f0
+
+m17n-db-devel_package() {
+	noarch=yes
+	depends="${makedepends} ${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/bin
```
Comment (Vaelatern) [inline srcpkgs/m17n-db/template]: What is this program for?

File: srcpkgs/m17n-db/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'm17n-db'
+pkgname=m17n-db
+version=1.7.0
+revision=1
+noarch=yes
+only_for_archs="aarch64 armv7hf i686 x86_64"
+build_style=gnu-configure
+short_desc="Multilingual text processing library - database"
```
Comment (Vaelatern) [inline srcpkgs/m17n-db/template]: Remove the `-` here

File: srcpkgs/m17n-db/template (hunk 1 above)
Comment (Vaelatern) [inline srcpkgs/m17n-db/template]: Probably want only the `noarch` line

**Comment by hyonhyoro** [comment]
The m17n-db is needed to build the uim, a multilingual input method framework, with the m17n-lib. Ordinarily, it has noarch dependency but makedepends on glibc-locales. I put a limit on *-musl because I didn't know whether I could used them.

**Comment by asergi** [comment]
ab6d6be9931c9c5c854bfcaa181ba353a9831b75 (#7587)

**Pull request closed without merging**